ResXManager.Model/ResourceManagerExtensions.cs

```diff
@@ -44,12 +46,22 @@ public static IList<ProjectFile> GetAllSourceFiles([NotNull] this DirectoryInfo
                 var project = FindProject(directory, solutionFolder.FullName);
 
                 var projectName = "<no project>";
+                var assemblyName = "<no assembly>";

```

Comment:
This part does the loading for the standalone version only! The VSIX should behave the same.

Reply:
Also retrieving this during project load could cause performance issues. Maybe better get this information only when needed during export.

Reply:
Current implementation won't work with the new project formats, where there will be no entries for AssemblyName and RootNamespace if they match the default.